app/Models/Image.php

```diff
@@ -290,6 +290,11 @@ public function isFont(): bool
         return in_array($this->ext, ['woff', 'woff2']);
     }
 
+    public function isFile(): bool
+    {
+        return in_array($this->ext, ['pdf', 'gif', 'webp', 'pdf', 'xls', 'xlsx', 'csv', 'mp3', 'ogg', 'json']);
```

Comment:
bug gif and webp generate images. the name isFile also doesn't make sense. A font, and image, and an mp3 are all files. Better to name it hasThumbnail() checking for images, and doing `!hasThumbnail()` in the code